server/routes/visitJourney/dateAndTime.test.ts

```diff
@@ -481,3 +490,147 @@ testJourneys.forEach(journey => {
     })
   })
 })
+
+describe(`Update journey specific testing`, () => {
+  it('Restriction change: Open to closed, no visit slots available - visitor', () => {
```

Comment:
Very minor point on wording of test names (this and the other ones): I think rather than "no visit slots available" it's more accurate to say something like "no availability for the selected visit slot".

The difference is that we're using "visit slot" to mean distinct time and dates where there may (or may not!) be availability to schedule a visit.